examples/access_review/workflows/okta_password_policy_monitoring.py

```diff
@@ -0,0 +1,132 @@
+from typing import Annotated
+from datetime import datetime, timedelta
+import json
+
+from admyral.workflow import workflow, Schedule
+from admyral.typings import JsonValue
+from admyral.action import action, ArgumentMetadata
+from admyral.actions import get_okta_logs, send_slack_message_to_user_by_email
+
+
+@action(
+    display_name="Calculate Time Range for Last Full Hour",
+    display_namespace="Utilities",
+    description="Calculate the time range for the last full hour",
+)
+def calculate_time_range() -> tuple[str, str]:
```

Comment:
Function name could be more descriptive, e.g., `get_time_range_of_last_full_hour`.